pkg/sql/executor_statement_metrics.go

```diff
@@ -293,6 +293,13 @@ func (ex *connExecutor) recordStatementSummary(
 			ex.planner.DistSQLPlanner().distSQLSrv.Metrics.ContendedQueriesCount.Inc(1, dbName, appName)
 			ex.planner.DistSQLPlanner().distSQLSrv.Metrics.CumulativeContentionNanos.Inc(queryLevelStats.ContentionTime.Nanoseconds(), dbName, appName)
 		}
+
+		if queryLevelStats.CPUTime > 0 {
```

Comment:
Let's eliminate this check here. The check on contention is relevant, as if contention can be absent in query execution, and this absence is marked by the `ContentionTime > 0` condition.

If sampled (`queryLevelStatsOk`), sql cpu time cannot be absent in the same way, so we should be incrementing the `SQLCPUStatementsCount` metric always.